diff -ruN realtek/RadioPower.sh realtek/RadioPower.sh
--- realtek/RadioPower.sh	2010-05-07 02:31:32.000000000 +0000
+++ realtek/RadioPower.sh	2010-08-07 05:20:09.000000000 +0000
@@ -2,9 +2,9 @@
 
 if [ "$1" = "RFON" ]; then
 	dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Set string:org.freedesktop.NetworkManager string:WirelessEnabled variant:boolean:true
-	echo "==============>Now Polling Method Turn  RF ON !" > /etc/acpi/events/RadioPowerTest
+	echo "==============>Now Polling Method Turn  RF ON !" > /etc/realtec/events/RadioPowerTest
 else
 	dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Set string:org.freedesktop.NetworkManager string:WirelessEnabled variant:boolean:false
-	echo "==============>Now Polling Method Turn RF OFF !" > /etc/acpi/events/RadioPowerTest
+	echo "==============>Now Polling Method Turn RF OFF !" > /etc/realtec/events/RadioPowerTest
 fi
 
diff -ruN HAL/rtl8192/Makefile HAL/rtl8192/Makefile
--- HAL/rtl8192/Makefile        2010-08-16 17:51:23.622270739 +0000
+++ HAL/rtl8192/Makefile        2010-08-16 18:05:28.185520087 +0000
@@ -512,7 +512,8 @@

 KSRC := /lib/modules/$(KVER)/build
 INSTALL_PREFIX :=
-IMG_DIR := /lib/firmware/
+IMG_DIR := /lib/firmware/$(KVER)/RTL8192SE/
+#IMG_DIR := /lib/firmware/
 all: modules

 modules:

