diff -ruN mod/HAL/rtl8192/Makefile mod2/HAL/rtl8192/Makefile
--- mod/HAL/rtl8192/Makefile	2010-05-07 02:31:32.000000000 +0000
+++ mod2/HAL/rtl8192/Makefile	2010-08-07 05:15:28.039666473 +0000
@@ -517,7 +517,7 @@
 
 KSRC := /lib/modules/$(KVER)/build
 INSTALL_PREFIX :=
-IMG_DIR := /lib/firmware/
+IMG_DIR := /lib/firmware/$(KVER)/
 all: modules
 
 modules:
diff -ruN mod/realtek/RadioPower.sh mod2/realtek/RadioPower.sh
--- mod/realtek/RadioPower.sh	2010-05-07 02:31:32.000000000 +0000
+++ mod2/realtek/RadioPower.sh	2010-08-07 05:20:09.000000000 +0000
@@ -2,9 +2,9 @@
 
 if [ "$1" = "RFON" ]; then
 	dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Set string:org.freedesktop.NetworkManager string:WirelessEnabled variant:boolean:true
-	echo "==============>Now Polling Method Turn  RF ON !" > /etc/acpi/events/RadioPowerTest
+	echo "==============>Now Polling Method Turn  RF ON !" > /etc/realtec/events/RadioPowerTest
 else
 	dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Set string:org.freedesktop.NetworkManager string:WirelessEnabled variant:boolean:false
-	echo "==============>Now Polling Method Turn RF OFF !" > /etc/acpi/events/RadioPowerTest
+	echo "==============>Now Polling Method Turn RF OFF !" > /etc/realtec/events/RadioPowerTest
 fi
 
diff -ruN mod/HAL/rtl8192/Makefile mod2/HAL/rtl8192/Makefile
--- mod/HAL/rtl8192/Makefile	2010-05-07 02:31:32.000000000 +0000
+++ mod2/HAL/rtl8192/Makefile	2010-08-09 07:09:26.982100198 +0000
@@ -25,7 +25,6 @@
 		-DCONFIG_PM_RTL 			\
 		-DCONFIG_PM 				\
 		-DENABLE_IPS 				\
-		-DENABLE_LPS 				\
 		-DCONFIG_ASPM_OR_D3 			\
 		-DENABLE_GPIO_RADIO_CTL 		\
 		-DMP_DRIVER=0				\
@@ -60,7 +59,6 @@
 		-DCONFIG_PM 				\
 		-DENABLE_GPIO_RADIO_CTL 		\
 		-DRTL8190_Download_Firmware_From_Header=1 \
-		-DENABLE_LPS 				\
 		-DCONFIG_RX_CMD				\
 		-DENABLE_IPS 				\
 		-DUSE_FW_SOURCE_IMG_FILE 		\
@@ -111,7 +109,6 @@
 		-DEEPROM_OLD_FORMAT_SUPPORT=1		\
 		-DUSE_FW_SOURCE_IMG_FILE		\
 		-DENABLE_IPS				\
-		-DENABLE_LPS				\
 		-DENABLE_GPIO_RADIO_CTL			\
 		-DCONFIG_PM_RTL 			\
 		-DCONFIG_PM 				\
@@ -142,7 +139,6 @@
 		-DCONFIG_PM_RTL 			\
 		-DENABLE_GPIO_RADIO_CTL			\
 		-DENABLE_IPS 				\
-		-DENABLE_LPS				\
 		-DCONFIG_ASPM_OR_D3 			\
 		-DCUSTOMER_ID_INTEL_CMPC 		\
 		-DMESH_8192_SPECIFIED 			\
@@ -192,7 +188,6 @@
 		-DCONFIG_PM_RTL 			\
 		-DCONFIG_PM 				\
 		-DENABLE_IPS 				\
-		-DENABLE_LPS 				\
 		-DCONFIG_ASPM_OR_D3 			\
 		-DENABLE_GPIO_RADIO_CTL 		\
 		-DMP_DRIVER=0				\
@@ -629,7 +624,6 @@
 #CFLAGS += -DENABLE_GPIO_RADIO_CTL  
 #EXTRA_CFLAGS += -DCONFIG_PM
 #EXTRA_CFLAGS += -DENABLE_IPS
-#EXTRA_CFLAGS += -DENABLE_LPS
 #EXTRA_CFLAGS += -DCONFIG_ASPM_OR_D3
 #CFLAGS += -DENABLE_AMSDU
 #CFLAGS += -DADHOC_11N
@@ -654,7 +648,6 @@
 CFLAGS += -DENABLE_GPIO_RADIO_CTL  
 CFLAGS += -DRTL8190_Download_Firmware_From_Header=1 
 CFLAGS += -DENABLE_IPS
-CFLAGS += -DENABLE_LPS
 CFLAGS += -DCONFIG_ASPM_OR_D3
 CFLAGS += -D_RTL8192_EXT_PATCH_ \
 	  -DMESH_8192_SPECIFIED \
diff -ruN mod/rtllib/Makefile mod2/rtllib/Makefile
--- mod/rtllib/Makefile	2010-05-07 02:31:32.000000000 +0000
+++ mod2/rtllib/Makefile	2010-08-09 07:09:01.862102537 +0000
@@ -16,13 +16,11 @@
 	EXTRA_CFLAGS += -DRTL8192S_DISABLE_FW_DM=0
 	EXTRA_CFLAGS += -DRTL8192SE
 	EXTRA_CFLAGS += -DENABLE_IPS
-	EXTRA_CFLAGS += -DENABLE_LPS
 endif
 ifeq ($(NIC_SELECT), RTL8192SU)
 	EXTRA_CFLAGS += -DRTL8192S_DISABLE_FW_DM=0
 	EXTRA_CFLAGS += -DRTL8192SU
 	EXTRA_CFLAGS += -DENABLE_IPS
-	EXTRA_CFLAGS += -DENABLE_LPS
 	EXTRA_CFLAGS += -DUSB_RX_AGGREGATION_SUPPORT
 	EXTRA_CFLAGS += -DTCP_CSUM_OFFLOAD_RX
 endif
@@ -32,7 +30,6 @@
 ifeq ($(NIC_SELECT), RTL8192E)
 	EXTRA_CFLAGS += -DRTL8192E
         EXTRA_CFLAGS += -DENABLE_IPS
-        EXTRA_CFLAGS += -DENABLE_LPS
 endif
 ifeq ($(NIC_SELECT),RTL8192U)
 EXTRA_CFLAGS += -DRTL8192U
@@ -52,7 +49,6 @@
 		-DRTL8192S_DISABLE_FW_DM=0 \
 		-D_RTL8192_EXT_PATCH_\
 		-DENABLE_IPS \
-		-DENABLE_LPS \
 		-DCUSTOMER_ID_INTEL_CMPC
 ifeq ($(AUTHTYPE),SAE)
 EXTRA_CFLAGS += -DMESH_ABBR_HANDSHK_SUPPORT\
